lib/internal/modules/esm/loader.js

```diff
@@ -253,20 +258,27 @@ class ESMLoader {
    * @returns {string}
    */
   getBaseURL(url) {
+    let baseURL = this.baseURLs.get(url);
+    if (typeof baseURL === 'string') {
+      return baseURL;
+    }
```

Comment:
Could you please add a comment explaining why would or wouldn't it be a string?

Reply:
tried to explain it